src/main/java/ch/njol/skript/expressions/ExprExperienceAmount.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.ExpBottleEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Experience Amount")
+@Description("The amount of experience released in an experience bottle hit event.")
+@Examples({"on experience bottle hit:",
+	"\texperience amount is below 5",
+	"\tincrease experience amount by 3"})
+@Since("INSERT VERSION")
+@Events("Experience Bottle Hit")
+public class ExprExperienceAmount extends SimpleExpression<Integer> {
```

Comment:
This should be `SimpleExpression<Number>`. Integer should also be changed to Number in other places, such as getReturnType.

Reply:
`ExpBottleEvent#getExperience()` & `ExpBottleEvent#setExperience()` return and accept only an Integer. That's why I used Integer. Should I still change this?

Reply:
Yes